models/snowflake/sessions_by_customer_id.sql

```diff
@@ -0,0 +1,43 @@
+SELECT
```

Comment:
This is sort of related to my above comment, and yes it is to follow Google's conventions. 

It's confusing, there are kind've two notions of a session - the GA type of session where if you change your marketing attribution you start a new session automatically, and the snowplow session, where a new channel doesn't mean a new session is created (unless 30min of inactivity). 

So in this model, we are returning one page view per session if they are not considering intrasession channels. We are returning more than one page view per session if they are considering intrasession channels - in this case, each event  ‘page view’ is considered to be a separate session and is treated as such. So, the rows returned by this model are associated with page views, but we don’t treat them as page_views, we treat them as separate sessions, in the GA sense. Mike kept consistency of naming to make it more of a reproduction of the fractribution stuff. 

Let me know if this is still confusing, Mike also explained it to me (he wrote the bulk of this code) so I may not have explained it well enough!

Reply:
Thank you very much for the detailed explanation, it is much clearer! One thing that I wanted to mention earlier but forgot is that we do not keep the detailed documentation inside the models anymore but shifted anything that are not the absolute essentials to https://github.com/snowplow/documentation. We are currently changing the structure slightly within the data modelling sections, but what I can do is once that is finalised in a day or two I will open a new draft PR, create a placeholder folder with the related index.md file and copy the currently existing readme contents there. Once there, perhaps it would make sense to explain anything special that the users will need to know about understanding the model and how to operate it (e.g. custom, more advanced used cases). Would you be happy to add such extra documentation and / or review the PR together?

Reply:
Yes of course, thanks for your help!